src/Services/Identity/Identity.Application/Common/Exceptions/ValidationException.cs

```diff
@@ -0,0 +1,21 @@
+﻿using FluentValidation.Results;
+
+namespace Identity.Application.Common.Exceptions;
+
+public class ValidationException : Exception
+{
+    public ValidationException()
+            : base("One or more validation failures have occurred.")
+    {
+        Errors = new Dictionary<string, string[]>();
+    }
+
+    public ValidationException(IEnumerable<ValidationFailure> failures)
+        : this()
+    {
+        Errors = failures
+            .GroupBy(e => e.PropertyName, e => e.ErrorMessage)
+            .ToDictionary(failureGroup => failureGroup.Key, failureGroup => failureGroup.ToArray());
+    }
+    public IDictionary<string, string[]> Errors { get; }
```

Comment:
When describing a class, use the following structure:
· Declare all class members, including readonly or constants
· Declare all public properties, including auto-properties
· Declare all constructor(s)
· Declare the destructor, if any
· Declare all public methods
· Declare all protected methods
· Declare all private methods